src/twisted/newsfragments/9463.feature

```diff
@@ -0,0 +1 @@
+twisted.internet._sslverify now uses SHA256 instead of MD5 for ssl session identifier generation by default.
```

Comment:
Public names are preferred for news fragments.  The subject should be `twisted.internet.ssl.CertificateOptions`, I think.